google/cloud/internal/backoff_policy.h

```diff
@@ -132,6 +132,9 @@ class ExponentialBackoffPolicy : public BackoffPolicy {
         maximum_delay_(std::chrono::duration_cast<std::chrono::microseconds>(
             maximum_delay)),
         scaling_(scaling) {
+    if (current_delay_range_ > maximum_delay_) {
+      current_delay_range_ = maximum_delay_;
+    } 
```

Comment:
Could we move this into the initialization (and move "current" down in the member list) with:
```
  current_delay_range_((std::min)(2 * initial_delay_, maximum_delay_))
```

And then (unrelated to this PR, but perhaps addressable here), it seems like just `current_delay_` would be preferable to `current_delay_range_`.